scripts/build.gradle

```diff
@@ -2,6 +2,8 @@ plugins {
   id 'local.java.lint-shellcheck-conventions'
 }
 
+tasks.named("shellcheck").configure { dependsOn("spotlessGroovyGradle") }
```

Comment:
Why was this line added?

Reply:
so it was actually added for the exact reason that you discovered below. there was an implicit dependency in the task in version 7, but those have been disallowed in version 8. so i'll add that to fix that error as well

Reply:
This would be a good thing to announce at our next engineering meeting and explain how to fix, in case anyone runs a gradle task that you haven't tested.